<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>484</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_1">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>11</y>
          <width>361</width>
          <height>361</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QListWidget" name="lw_modes">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>110</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="midLineWidth">
            <number>0</number>
           </property>
           <property name="isWrapping" stdset="0">
            <bool>false</bool>
           </property>
           <property name="uniformItemSizes">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>static color</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>audio lavel</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>fire</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QStackedWidget" name="sw_mode">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="page">
              <widget class="QDial" name="gain">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>120</y>
                 <width>81</width>
                 <height>81</height>
                </rect>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="value">
                <number>200</number>
               </property>
               <property name="tracking">
                <bool>true</bool>
               </property>
               <property name="invertedAppearance">
                <bool>true</bool>
               </property>
               <property name="invertedControls">
                <bool>false</bool>
               </property>
              </widget>
              <widget class="QProgressBar" name="right">
               <property name="geometry">
                <rect>
                 <x>30</x>
                 <y>50</y>
                 <width>118</width>
                 <height>23</height>
                </rect>
               </property>
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="value">
                <number>24</number>
               </property>
              </widget>
              <widget class="QProgressBar" name="general">
               <property name="geometry">
                <rect>
                 <x>30</x>
                 <y>20</y>
                 <width>118</width>
                 <height>23</height>
                </rect>
               </property>
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="value">
                <number>24</number>
               </property>
              </widget>
              <widget class="QProgressBar" name="left">
               <property name="geometry">
                <rect>
                 <x>30</x>
                 <y>80</y>
                 <width>118</width>
                 <height>23</height>
                </rect>
               </property>
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="value">
                <number>24</number>
               </property>
              </widget>
             </widget>
             <widget class="QWidget" name="page_2">
              <widget class="QPushButton" name="set_color_pb">
               <property name="geometry">
                <rect>
                 <x>40</x>
                 <y>40</y>
                 <width>151</width>
                 <height>23</height>
                </rect>
               </property>
               <property name="text">
                <string>set color</string>
               </property>
              </widget>
             </widget>
             <widget class="QWidget" name="page_5"/>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="setAlMod">
             <property name="text">
              <string>Set</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QListWidget" name="listWidget">
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Srial Port</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Audio in</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page_3">
           <widget class="QGroupBox" name="groupBox">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>10</y>
              <width>211</width>
              <height>111</height>
             </rect>
            </property>
            <property name="title">
             <string>Serial Port Options</string>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Port:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="com_ports"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Baudrate:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="com_speeds"/>
             </item>
             <item row="2" column="0">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>Re-Scan Ports</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="page_4">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="geometry">
             <rect>
              <x>9</x>
              <y>-1</y>
              <width>211</width>
              <height>161</height>
             </rect>
            </property>
            <property name="title">
             <string>Audio in</string>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>input device</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="comboBox"/>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="pb_connect">
        <property name="text">
         <string>Connect</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_disconnect">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Disconnect </string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
